src/App.php

```diff
@@ -335,7 +335,14 @@ public function __construct(Database $database, IManageConfigValues $config, App
 	 */
 	protected function load(DbaDefinition $dbaDefinition, ViewDefinition $viewDefinition)
 	{
-		set_time_limit(0);
+		if ($this->config->get('system', 'ini_max_execution_time') !== false) {
+			set_time_limit((int)$this->config->get('system', 'ini_max_execution_time'));
+		}
+
+		// This has to be quite large to deal with embedded private photos
```

Comment:
I don't think this comment is necessary now that there's a config key with the associated comment.
```suggestion
```